packages/react-core/src/components/Switch/examples/Switch.md

```diff
@@ -176,19 +176,9 @@ import { Switch } from '@patternfly/react-core';
     aria-label="Message when on"
```

Comment:
Should we also update the aria-label when appropriate to demonstrate the intention? it's possible that, when it's uncontrolled, there are a few extra considerations to make to make sure the aria-label is meaningful. Like, what should we have the aria-label say if it wont change when the switch is on/off?

Reply:
I opened issue #7468 for this issue as well as an issue I noticed with the `aria-labelledby` attribute.